src/caffe/layers/cudnn_conv_layer.cu

```diff
@@ -1,126 +1,182 @@
 #ifdef USE_CUDNN
 #include <vector>
 
+#include "caffe/filler.hpp"
 #include "caffe/layers/cudnn_conv_layer.hpp"
+#include "caffe/util/gpu_memory.hpp"
+#include "caffe/util/im2col.hpp"
+#include "caffe/util/math_functions.hpp"
+
+// Those defines serve single purpose to keep sane C++ formatting
+// in presence of <80 characters rule
+#define cudnnConvFwd                       cudnnConvolutionForward
+#define cudnnConvBwdBias                   cudnnConvolutionBackwardBias
+#define cudnnConvBwdFilter                 cudnnConvolutionBackwardFilter
+#define cudnnConvBwdData                   cudnnConvolutionBackwardData
```

Comment:
Is this really necessary? If you think so, please remove the huge floating space in the middle.